node/cli/src/command/chain_setup.rs

```diff
@@ -78,6 +79,11 @@ impl BootstrapChainCmd {
 				other
 			)),
 		};
+		let network_id = if config.n.len() <= 64 {
```

Comment:
Maybe for now just set `let network_id = config.network_id.clone()`. We can add a check later.